app/navigation/Navigator.scala

```diff
@@ -92,20 +92,23 @@ class Navigator @Inject()() {
       case  _  => routes.RepaymentTypeController.onPageLoad(NormalMode)
     }
 
-  private def getEntryDetails(answers: UserAnswers): Call = answers.get(CustomsRegulationTypePage) match {
-    case Some(CustomsRegulationType.UnionsCustomsCodeRegulation)  => {
-      answers.get(NumberOfEntriesTypePage).contains(NumberOfEntriesType.Single) match {
-        case true => routes.ArticleTypeController.onPageLoad (NormalMode)
-        case _=> routes.BulkFileUploadController.showFileUpload
-      }
+  private def getEntryDetails(answers: UserAnswers): Call =
```

Comment:
Can we refactor this? Maybe just match on answers.get(CustomsRegulationTypePage) and check the CustomsRegulationType. 'contains' is good for checkboxes.

Reply:
@lmugnai  is this comment specifically related to get getEntryDetails?
![image](https://user-images.githubusercontent.com/58818306/109667414-04117400-7b68-11eb-9faf-24e468d5d0b4.png)